app/src/main/res/layout/fragment_example.xml

```diff
@@ -1,45 +1,111 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:padding="@dimen/spacing_large"
     android:gravity="center"
     android:orientation="vertical">
 
-    <TextView
+    <EditText
+        android:id="@+id/vMailInput"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/spacing_none"
-        android:layout_weight="1"
-        android:layout_margin="@dimen/spacing_medium_more"
-        android:gravity="top|center_horizontal"
-        android:text="@string/example_title"
-        android:textSize="@dimen/font_medium_extra" />
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:gravity="center"
+        android:hint="Mail"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.469" />
+
+    <ImageView
+        android:id="@+id/imageViewFoto"
+        android:layout_width="413dp"
+        android:layout_height="208dp"
```

Comment:
Maybe we can use a guideline to define the size of the photo.

You can read more here: https://riggaroo.co.za/constraintlayout-guidelines-barriers-chains-groups/